### DISCUSSION TITLE
[IntelliSense] Support wildcard in tailwindCSS.classAttributes

### DISCUSSION THREAD

**Discussion by phuctm97** [tailwindlabs/tailwindcss#9451] [Ideas]
To enable use cases where you define custom class attributes or shared class name constants. Instead of having to list all class name "customClassName", "tooltipClassName", "buttonClassName", etc, configure "*ClassName".

**Comment by cmngan** [comment]
Related to https://github.com/tailwindlabs/tailwindcss-intellisense/issues/350
